executorCwl/src/main/scala/dx/executor/cwl/CwlTaskExecutor.scala

```diff
@@ -373,6 +373,9 @@ case class CwlTaskExecutor(tool: Process,
     updatedSource
   }
 
+  private lazy val rawStep: Option[String] = {
+    targetStep.map(_.split('/').grouped(2).flatMap(_.take(1)).toArray.mkString("/"))
```

Comment:
the `targetwf#targetStep` is in the format of `wf#step1/nested/step2`. But `cwltool --single-process` only accept step id sequences like `step1/step2`, so we need to drop the workflow ids.